### ISSUE TITLE
Error : GLFW

### ISSUE THREAD

**Issue by AnthonyJacob** [JuliaGL/GLVisualize.jl#131]
I am using Fedora 25

================================[ ERROR: GLFW ]=================================

LoadError: failed process: Process(`cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/home/Anthony/.julia/v0.5/GLFW/deps/usr -DGLFW_BUILD_DOCS=OFF -DGLFW_BUILD_EXAMPLES=OFF -DGLFW_BUILD_TESTS=OFF .`, ProcessExited(1)) [1]
while loading /home/Anthony/.julia/v0.5/GLFW/deps/build.jl, in expression starting on line 52

================================================================================
INFO: Building FreeType

================================[ BUILD ERRORS ]================================

WARNING: GLFW had build errors.

 - packages with build errors remain installed in /home/Anthony/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("GLFW")`
 - build a single package by running its `deps/build.jl` script

================================================================================

**Comment by fhenneke** [comment]
I just got the same error on Debian 8.

Does it also show something like `The RandR library and headers were not found` for you (a few lines above what you posted)?

**Comment by AnthonyJacob** [comment]
Nope, could not find 'vulkan' and 'x11' instead. I have updated my first message.

**Comment by SimonDanisch** [comment]
Well do you have a display manager like X11 or opengl installed?

**Comment by fhenneke** [comment]
I opened a new issue for my problem.
